addons/website/static/src/builder/plugins/options/avatars_option.xml

```diff
@@ -3,133 +3,61 @@
 
 <t t-name="website.AvatarsOption">
     <BuilderRow label.translate="Size">
-        <BuilderRange
-            styleAction="'--avatars-size'"
-            min="2"
-            max="6"
-            step="0.1"
-            displayRangeValue="false"
-            unit="'rem'"
-        />
+        <BuilderRange styleAction="'--avatars-size'" min="2" max="6" step="0.1" unit="'rem'"/>
     </BuilderRow>
     <BuilderRow label.translate="Order">
```

Comment:
Nitpick: add a space between the two `BuilderRow` (for consistency in the file)
(this a real a nitpick, I say it only because I added other comments haha, otherwise I wouldn't have mentioned it)